### ISSUE TITLE
Initialize in context learning approach in long context model

### ISSUE THREAD

**Issue by zprintz** [vanderbilt-data-science/pga-tour-advanced-analytics#17]

**Comment by zprintz** [comment]
I have compiled all data for the 2024 tournaments and have formatted it in the identical manner as what we have for the 2023 data.

I want to try the in context learning approach with all of the 2023 data as training data, asking the model to predict tournaments in 2024; however, the data is too long. I tried in Claude, and it says the message is 309% too long. Additionally, GPT4 generates an error saying the message is too long.

**Comment by zprintz** [comment]
Comprehensive 2023 data has been given to Professor Spencer-Smith. He will use it to predict 2024 tournaments through in context learning.

**Comment by zprintz** [comment]
I used Gemini to predict 2024 tournaments, but am confused as to how to continue the conversation.

I produced results for the first tournament, which resulted in an NDCG of 0.56. With the NDCG of chance being 0.60, this is a disappointing result; however, no conclusions can be drawn until additional tournaments are predicted.

https://console.cloud.google.com/vertex-ai/generative/multimodal/locations/us-central1/prompts/8991011145045245952?project=gleaming-nomad-419821&pli=1
